### PR TITLE
Update sarek specific UPPMAX config

### PR DESCRIPTION
**Pull request by MaxUlysse**
- Update sarek specific UPPMAX config

### CODE REVIEW

File: conf/pipeline/sarek/uppmax.config [hunk 1]
```diff
@@ -3,10 +3,12 @@
...
-}
 
-params {
-  // Just useful until iGenomes is updated on UPPMAX
+// Just useful until iGenomes is updated on UPPMAX
   igenomeIgnore = true
   genomes_base = params.genome == 'GRCh37' ? '/sw/data/uppnex/ToolBox/ReferenceAssemblies/hg38make/bundle/2.8/b37' : '/sw/data/uppnex/ToolBox/hg38bundle'
 }
+
+if (hostname ==~ "i.*") {
```
Comment (alneberg) [inline conf/pipeline/sarek/uppmax.config]: I think you should add this for hostname r.* and b.* as well?

Comment (MaxUlysse) [reply conf/pipeline/sarek/uppmax.config]: If I remember well `r.*` and `b.*` have a similar settings, I'll check what we used to give with the old scilifelab/sarek repo

File: conf/pipeline/sarek/uppmax.config [hunk 2]
```diff
@@ -3,10 +3,12 @@
 params {
   config_profile_contact = 'Maxime Garcia (@MaxUlysse)'
   config_profile_description = 'nf-core/sarek uppmax profile provided by nf-core/configs'
-}
```
Comment (MaxUlysse) [inline conf/pipeline/sarek/uppmax.config]:
```suggestion

    singleCPUmem = 7.GB
```

File: conf/pipeline/sarek/uppmax.config [hunk 3]
```diff
@@ -3,10 +3,13 @@
...
 
-params {
-  // Just useful until iGenomes is updated on UPPMAX
+    singleCPUmem = 7.GB
```
Comment (MaxUlysse) [inline conf/pipeline/sarek/uppmax.config]:
```suggestion
  singleCPUmem = 7.GB
```

File: conf/pipeline/sarek/uppmax.config (hunk 1 above)
Comment (MaxUlysse) [reply conf/pipeline/sarek/uppmax.config]: we had that:
https://github.com/SciLifeLab/Sarek/blob/master/conf/uppmax-slurm.config#L13

Comment (alneberg) [reply conf/pipeline/sarek/uppmax.config]: Might want to have that in Megabytes to be sure that you avoid problems with the GB to MB conversion.

Comment (MaxUlysse) [reply conf/pipeline/sarek/uppmax.config]: I think that's why we have 7 and not 8 GB

File: conf/pipeline/sarek/uppmax.config [hunk 4]
```diff
@@ -3,10 +3,13 @@
 params {
   config_profile_contact = 'Maxime Garcia (@MaxUlysse)'
   config_profile_description = 'nf-core/sarek uppmax profile provided by nf-core/configs'
-}
 
-params {
-  // Just useful until iGenomes is updated on UPPMAX
+  singleCPUmem = 7.GB
```
Comment (alneberg) [inline conf/pipeline/sarek/uppmax.config]:
```suggestion
  singleCPUmem = 7000.MB
```

File: conf/pipeline/sarek/uppmax.config [hunk 5]
```diff
@@ -3,10 +3,13 @@
...
-}
 
-params {
-  // Just useful until iGenomes is updated on UPPMAX
+  singleCPUmem = 7000.MB
+// Just useful until iGenomes is updated on UPPMAX
   igenomeIgnore = true
   genomes_base = params.genome == 'GRCh37' ? '/sw/data/uppnex/ToolBox/ReferenceAssemblies/hg38make/bundle/2.8/b37' : '/sw/data/uppnex/ToolBox/hg38bundle'
 }
+
```
Comment (MaxUlysse) [inline conf/pipeline/sarek/uppmax.config]:
```suggestion
 if (hostname ==~ "r.*") {
  params.singleCPUmem = 6400.MB
}
```

**Pull request merged**